molecule/shared/substrate/lxd/create.yml

```diff
@@ -0,0 +1,84 @@
+---
+- name: Manage LXD containers
+  hosts: localhost
+  gather_facts: false
+  connection: local
+  vars:
+    molecule_inventory:
+      molecule:
+        hosts: {}
+    molecule_yml:
+      platforms:
+        - name: instance
+          image: ubuntu:20.04
+          user: ansible
+
+  tasks:
+    - name: Ensure LXD is installed
+      ansible.builtin.command:
+        cmd: snap install lxd
+      become: true
+
+    - name: Initialize LXD (if not already initialized)
+      command: lxd init --auto
+      args:
+        creates: /var/lib/lxd/lxd.db
+      become: true
+
+    - name: Check if LXD container exists
+      command: lxc list "{{ platform['name'] }}" --format=json
+      register: lxd_container_info
+      ignore_errors: true
+      loop: "{{ molecule_yml['platforms'] }}"
+      loop_control:
+        loop_var: platform
+
+    - name: Debug container existence
+      ansible.builtin.debug:
+        var: lxd_container_info
+
```

Comment:
You should ensure that the `cloud-init` file is being generated for the local environment based on the collections template. Like is done here

https://github.com/deeep-network/bedrock/blob/4f423edd99283be38b6f3f63625086a29009487d/molecule/shared/substrate/orbstack/create.yml#L11-L15